compiler/rustc_mir_transform/src/shim.rs

```diff
@@ -74,11 +74,13 @@ fn make_shim<'tcx>(tcx: TyCtxt<'tcx>, instance: ty::InstanceDef<'tcx>) -> Body<'
                 let mut body = EarlyBinder::bind(body.clone()).instantiate(tcx, args);
                 debug!("make_shim({:?}) = {:?}", instance, body);
 
-                // Run empty passes to mark phase change and perform validation.
                 pm::run_passes(
                     tcx,
                     &mut body,
-                    &[],
+                    &[
```

Comment:
Does this mean that generator drop shims will inherit the panic strategy of the crate that they're being monomorphized in, rather than the crate that they're coming from?

Reply:
Huh. Do you know whether other shims default to definition-crate or mono-crate for their panic strategy? This inconsistency seems important to iron out, I feel? 🤔